cli.py

```diff
@@ -48,8 +49,27 @@ def attr_upd():
         return
 
     if args.list:
-        print(appliance_manager.aircons)
-        print(appliance_manager.washer_dryers)
+        for index, cooler in enumerate(appliance_manager.aircons):
+            print(index, cooler)
+        for index, hole in enumerate(appliance_manager.washer_dryers):
+            print(index, hole)
+        applianceelementinput = input("Enter 0-" + str(index) + " for appliance details, otherwise type 'n': ")
+        try:
+            applianceelement = int(applianceelementinput)
```

Comment:
```suggestion
        appliance_input = input("Enter 0-" + str(index) + " for appliance details, otherwise type 'n': ")
        try:
            selected_appliance = int(appliance_input)
```

Reply:
we should use lower_case_with_underscores for names pls